sig-contributor-experience/charter.md

```diff
@@ -1,58 +1,143 @@
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+The [Contributor Experience Special Interest Group] (SIG) is responsible for improving the experience of those who upstream contribute to the Kubernetes project. We do this by creating, and maintaining programs and processes that promote community health and reduce project friction, while retiring those programs and processes that don't. Being conscientious of our contributor base is critical to scaling the project, growing the ecosystem, and helping the project succeed.
+
+We do this by listening - whether it’s through our roadshows to SIG meetings, surveys, data, or [GitHub issues], we take in the feedback and turn it into our [project list]. We build a welcoming and inclusive community of contributors by giving them places to be heard and productive.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Establish policies, standards and procedures for the use, [moderation], and management of all public platforms officially used by the project, including but not limited to:
+  - [discuss.kubernetes.io]
+  - [GitHub]
+  - [Mailing lists] / Google groups for the project as a whole (eg: kubernetes-dev@googlegroups.com) and for individual sigs and wgs where the Chairs have provided us ownership
+  - [Slack]
+  - [/kubernetescommunity] YouTube channel
+  - [Zoom]
+- Establish and staff teams responsible for the administration and moderation of these platforms
+  - Teams must be staffed by trusted contributors spanning time zones, see [moderation] for more detail
+  - They are authorized to take immediate action when dealing with code of conduct issues, see [moderators] for the full list
+  - They are expected to escalate to the project's code of conduct committee when issues rise above the level of simple moderation
+- Work with other SIGs and interested parties in the project to execute GitHub tasks where required, see [GitHub] for more detail
+- Own and execute events that are targeted to the Kubernetes contributor community, including:
+  - The weekly [Kubernetes Community meeting]
+  - [Contributor Summit(s)]
+  - [Steering Committee elections] (though we do not own policy creation, see 'out of scope' below)
+  - Retrospective moderation for other SIGs upon request
+  - Other events, like other SIG face to face events, upon request and consideration
+- Strategize, build, and execute on scalable [mentoring programs] for all contributor levels. These may include:
+  - [Google Summer of Code]
+  - [Outreachy]
+  - [Meet Our Contributors]
+  - [Group Mentoring - WIP]
+  - [The 1:1 Hour - WIP]
+  - Speed Mentoring at KubeCon
+We supply the moderation teams that span time zones for 24/7 coverage. This includes immediate action when dealing with code of conduct issues in a public spaces
```

Comment:
- Is this for the speed mentoring session or in general? If in general, this needs to go as a sub-point under moderation, or else it renders with the Speed Mentoring point.
- Maybe we can rephrase this as `We supply moderation teams spanning multiple time zones for 24/7 coverage.`?
- s/a public spaces/public places.

Reply:
fixed